auth/auth.go

```diff
@@ -249,16 +250,34 @@ type Options2LO struct {
 	UseIDToken bool
 }
 
-func (c *Options2LO) client() *http.Client {
-	if c.Client != nil {
-		return c.Client
+func (o *Options2LO) client() *http.Client {
+	if o.Client != nil {
+		return o.Client
 	}
 	return internal.CloneDefaultClient()
 }
 
+func (o *Options2LO) validate() error {
+	if o == nil {
+		return errors.New("auth: options must be provided")
```

Comment:
Would it make sense to add "Options2LO" to these error messages? For example: ""auth: Options2LO.Email must be provided", etc?

Reply:
I don't think it is necessary. The only time a user would see these is if you call the New method for 2lo token source directly. I think it is implied what these all relate to for that reason.